boards/redboard_artemis_nano/src/main.rs

```diff
@@ -148,11 +148,11 @@ pub unsafe fn main() {
     pwr_ctrl.enable_iom2();
 
     // Enable PinCfg
-    &peripherals
+    let _ = &peripherals
```

Comment:
I'll probably get an obvious answer, but why is this necessary?

Reply:
Example:
```
warning: unused borrow that must be used
   --> boards/redboard_artemis_nano/src/main.rs:233:5
    |
233 |     &peripherals.iom2.enable();
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ the borrow produces a value
    |
help: use `let _ = ...` to ignore the resulting value
    |
233 |     let _ = &peripherals.iom2.enable();
    |     +++++++
```